pkg/apis/flowcontrol/types.go

```diff
@@ -0,0 +1,278 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	api "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+)
+
+const (
+	// APIGroupAll matches all api groups.
+	APIGroupAll = "*"
```

Comment:
where are these used?

Reply:
Since we are now using the rule types imported from `rbac` we do not need our own copy of these constants.

Reply:
Let's remove them, then.

Reply:
I agree.

Reply:
removed